clients/web/src/lib/components/abstract/Toast/types.ts

```diff
@@ -0,0 +1,18 @@
+import type {IconSource} from "@steeze-ui/svelte-icon/types";
```

Comment:
Name this file `toast.types.ts` so its easier to jump to in the project

Reply:
That would be inconsistent with other type files; you can also use `toast types.ts` or `toast/types.ts` in vs code to jump to the file; shouldn't be an issue.